en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/enabling-cors.md

```diff
@@ -23,15 +23,27 @@ x-wso2-cors:
   accessControlAllowCredentials: true
 ```
 
-### Enable CORS configuration for endpoints
+If you are following the [Deploy API via APIM]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect/#via-api-manager) approach, you can add **CORS** configurations for each API using the [APIM]({{base_path}}/design/advanced-topics/enabling-cors-for-apis/#EnablingCORSPerAPI).
 
-You can enable CORs for /authorize, /revoke, /token, /apikey, /userinfo and /health endpoints by enabling the following configuration in `toolkit-config.toml` located in `<MICRO-GW_TOOLKIT_HOME>/conf/.`
+### Enable CORS configurations globally
+
+You can enable CORS for Choreo Connect by configuring in the `config.toml` file, which is located in the `CHOREO-CONNECT_HOME/docker-compose/choreo-connect/conf/` directory.
+
+Follow the instructions below to enable CORS globally. Once this is enabled, it will apply this configurations through all endpoints and APIs deployed in Choreo Connect.
+
+1. Open the `CHOREO-CONNECT_HOME/docker-compose/choreo-connect/conf/config.toml` file.
+2. Locate the following configuration set and make the `enabled` attribute to `true` with the required CORS attributes there.
```

Comment:
Let's put a note saying that if cors for a certain API is disabled from API Level Configurations, the global Configurations will apply.

Reply:
Added a note